templates/block-templates/single-certificate.html

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Is the leading space needed for some reason?

Reply:
no, typo, thanks.